### PR TITLE
chore: update project for app settings configuration first

### PR DESCRIPTION
**Pull request by WilliamBergamin**
### Type of change (place an x in the [ ] that applies)

- [x] New sample
- [ ] New feature
- [ ] Bug fix
- [ ] Documentation

### Summary

This PR introduces changes to make this sample `app settings` first rather then supporting the `cli`.

Once `cli` support for bolt is ironed out we can reintroduce this feature. 

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Bolt for Python Custom Function Template
 
-This is a Bolt for Python template app used to build custom functions in [Workflow Builder](https://api.slack.com/start#workflow-builder).
+This is a Bolt for Python template app used to build custom functions in
```
Comment (srajiang) [inline README.md:3]:
```suggestion
This is a Bolt for Python template app used to build custom functions for use in in
```

File: README.md [hunk 2]
```diff
@@ -9,73 +10,96 @@ you have permission to install apps. **Please note that the features in this
...
 [a Slack paid plan](https://slack.com/pricing).**
 
-### Install the Slack CLI
+### Developer Program
 
-To use this template, you need to install and configure the Slack CLI.
-Step-by-step instructions can be found in our
-[Quickstart Guide](https://api.slack.com/automation/quickstart).
+Join the [Slack Developer Program](https://api.slack.com/developer-program) for
+exclusive access to beta features, tooling, and resources created to help
```
Comment (srajiang) [inline README.md:16]:
```suggestion
exclusive access to sandbox environments for building and testing your apps, beta features, tooling, and resources created to help
```

File: README.md [hunk 3]
```diff
@@ -9,73 +10,96 @@ you have permission to install apps. **Please note that the features in this
...
-# Clone this project onto your machine
-$ slack create my-app -t slack-samples/bolt-python-custom-function-template
+1. Open [https://api.slack.com/apps/new](https://api.slack.com/apps/new) and
+   choose "From an app manifest"
+2. Choose the workspace you want to install the application to
+3. Copy the contents of [manifest.json](./manifest.json) into the text box that
+   says `*Paste your manifest code here*` (within the JSON tab) and click _Next_
+4. Review the configuration and click _Create_
+5. Click _Install to Workspace_ and _Allow_ on the screen that follows. You'll
+   then be redirected to the App Configuration dashboard.
```
Comment (srajiang) [inline README.md:30]:
```suggestion
   then be redirected to the App Settings dashboard.
```

File: README.md [hunk 4]
```diff
@@ -9,73 +10,96 @@ you have permission to install apps. **Please note that the features in this
...
-```zsh
-# Clone this project onto your machine
-$ slack create my-app -t slack-samples/bolt-python-custom-function-template
+1. Open [https://api.slack.com/apps/new](https://api.slack.com/apps/new) and
+   choose "From an app manifest"
+2. Choose the workspace you want to install the application to
+3. Copy the contents of [manifest.json](./manifest.json) into the text box that
+   says `*Paste your manifest code here*` (within the JSON tab) and click _Next_
+4. Review the configuration and click _Create_
+5. Click _Install to Workspace_ and _Allow_ on the screen that follows. You'll
```
Comment (srajiang) [inline README.md:29]: The button now doesn't always say Install to Workspace.

```suggestion
5. Click the _Install_ button and _Allow_ on the screen that follows. You'll
```

File: README.md [hunk 5]
```diff
@@ -9,73 +10,96 @@ you have permission to install apps. **Please note that the features in this
...
+ruff check
 
-# Run black from root directory (code formatting)
-$ black .
+# Run ruff from root directory for code formatting
+ruff format
+ruff check --fix
 ```
 
 ## Using Functions in Workflow Builder
```
Comment (srajiang) [inline README.md:83]:
```suggestion
## Using Steps in Workflow Builder
```

**Review by srajiang — COMMENTED** [review]
Left some wording comments/tweaks 🙇

File: README.md [hunk 6]
```diff
@@ -9,73 +10,96 @@ you have permission to install apps. **Please note that the features in this
...
-$ cd my-app
+### Environment Variables
 
-# Install dependencies
-$ pip install -r requirements.txt
-```
+Before you can run the app, you'll need to store some environment variables.
 
-## Running Your Project Locally
+1. Open your apps configuration page from this list, click **OAuth &
```
Comment (srajiang) [inline README.md:36]:
```suggestion
1. Open your apps setting page from this list, click **OAuth &
```

whoop found another!

**Review by srajiang — APPROVED** [review]
Found one more app config mention but otherwise looks good!

File: README.md (hunk 6 above)
Comment (WilliamBergamin) [reply README.md:36]: Good catch 💯 I should probably update this in all our other samples 🤔

**Review by filmaj — APPROVED** [review]

**Pull request merged**